blog/_posts/2017-06-27-julia-0.6-release.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  Julia 0.6 Release Announcement
+author: The Julia Community
+---
+
+The Julia community is thrilled to announce the release of version 0.6.0 of the Julia language.
+With a sweeping overhaul of the type system and numerous improvements to syntax and to the
+standard library, 0.6.0 is the most transformative release yet.
+A comprehensive list of changes in this release is available in the main Julia repository in
+the [NEWS](https://github.com/JuliaLang/julia/blob/release-0.6/NEWS.md#julia-v060-release-notes)
+log.
+
+The 0.6 release line is now considered the stable line of releases and is recommended for most
+users, as it provides both language and API stability.
+As with previous versions, we will continue to update 0.6 with bugfixes, releasing subsequent
+0.6.x versions.
+Only critical bug fixes will be ported to 0.5 and releases older than 0.5 are now unmaintained.
+Work on new features is now taking place on the `master` branch for the 0.7 development cycle.
+The plan is for 0.7 to be released simultaneously with 1.0, which means that 0.6 is the last
```

Comment:
I think I wrote this down in a few places, discourse and the pr that went to 0.7-dev. the plan is that 1.0 won't have any new features or significant changes from 0.7 other than removing deprecations, and will follow very shortly after 0.7.